f2/src/utils/formatAnalystEmail.js

```diff
@@ -99,6 +100,7 @@ const formatIncidentInformation = (data) => {
   delete data.howdiditstart.howManyTimes
   delete data.howdiditstart.howDidTheyReachYou
   delete data.whatWasAffected.affectedOptions
+  delete data.howdiditstart.whenDidItStart
```

Comment:
We don't need to delete this here (it was removed in https://github.com/cds-snc/report-a-cybercrime/blob/master/f2/src/forms/defaultValues.js instead, essentially)